[2022-04-29 15:31:26,511] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:26:23.872433+00:00 [queued]>
[2022-04-29 15:31:26,530] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:26:23.872433+00:00 [queued]>
[2022-04-29 15:31:26,532] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 15:31:26,533] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-04-29 15:31:26,534] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 15:31:26,553] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): import_cat.check_presence_of_cat2> on 2022-04-29 15:26:23.872433+00:00
[2022-04-29 15:31:26,561] {standard_task_runner.py:52} INFO - Started process 3718 to run task
[2022-04-29 15:31:26,568] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cdn_daily', 'import_cat.check_presence_of_cat2', 'manual__2022-04-29T15:26:23.872433+00:00', '--job-id', '1688', '--raw', '--subdir', 'DAGS_FOLDER/cdn_daily.py', '--cfg-path', '/tmp/tmp1fhdk40p', '--error-file', '/tmp/tmp0oj2vzml']
[2022-04-29 15:31:26,570] {standard_task_runner.py:77} INFO - Job 1688: Subtask import_cat.check_presence_of_cat2
[2022-04-29 15:31:26,678] {logging_mixin.py:109} INFO - Running <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:26:23.872433+00:00 [running]> on host d5ad096ffea5
[2022-04-29 15:31:26,795] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cdn_daily
AIRFLOW_CTX_TASK_ID=import_cat.check_presence_of_cat2
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T15:26:23.872433+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-29T15:26:23.872433+00:00
[2022-04-29 15:31:26,833] {python.py:175} INFO - Done. Returned value was: None
[2022-04-29 15:31:26,852] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=cdn_daily, task_id=import_cat.check_presence_of_cat2, execution_date=20220429T152623, start_date=20220429T153126, end_date=20220429T153126
[2022-04-29 15:31:26,906] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-29 15:31:27,068] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
